core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/resourcebuffer/ResourceBufferSpec.scala

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.dsde.rawls.dataaccess.resourcebuffer
+
+import akka.http.scaladsl.model.headers.OAuth2BearerToken
+import org.broadinstitute.dsde.rawls.mock.MockResourceBufferDAO
+import org.broadinstitute.dsde.rawls.model.{GoogleProjectId, RawlsUserEmail, RawlsUserSubjectId, UserInfo}
+import org.broadinstitute.dsde.rawls.resourcebuffer.ResourceBufferService
+import org.scalatest.AsyncFlatSpec
+
+class ResourceBufferSpec extends AsyncFlatSpec {
+
+
+  val userInfo = UserInfo(RawlsUserEmail("owner-access"), OAuth2BearerToken("token"), 123, RawlsUserSubjectId("123456789876543212345"))
+  val resourceBufferDAO = new MockResourceBufferDAO
+  val resourceBufferService = new ResourceBufferService(resourceBufferDAO, userInfo)
+
+
+  "getGoogleProjectFromRBS" should "get a Google Project ID" in {
```

Comment:
I wasn't really sure if any unit tests would be useful / worth the effort but can gladly add more if someone has more ideas

Reply:
Greg's PR has a couple of tests that `createWorkspace` and `cloneWorkspace` can get a project from RBS that could be implemented

Reply:
Yeah, I would rather test this inside of another test, than give it its on test. (Adding something like `verifyResourceBufferService isCalled`